src/index.ts

```diff
@@ -4,7 +4,8 @@ import { init as rootControllerInit } from './controllers/rootController/rootCon
 
 const app = express();
 app.use(cors());
-
+app.use(express.json());
+app.use(express.urlencoded({extended: true}));
```

Comment:
What does this do?

Reply:
The bodyparser middleware was moved into express itself, parses json when it receives it.